default/stringtables/en.txt

```diff
@@ -15434,6 +15434,9 @@ Protection Focus
 GASEOUS_LABEL
 Gas Giant Environment
 
+MINDLESS_LABEL
+Mindless Species
+
```

Comment:
I accept suggestions about a better way to call this.
However, it seems to fit well into current Exobot's description.